Mathlib/Data/List/Rotate.lean

```diff
@@ -250,46 +240,82 @@ theorem zipWith_rotate_one {β : Type _} (f : α → α → β) (x y : α) (l :
   simp
 #align list.zip_with_rotate_one List.zipWith_rotate_one
 
-set_option linter.deprecated false in
-theorem nthLe_rotate_one (l : List α) (k : ℕ) (hk : k < (l.rotate 1).length) :
-    (l.rotate 1).nthLe k hk =
-      l.nthLe ((k + 1) % l.length) (mod_lt _ (length_rotate l 1 ▸ k.zero_le.trans_lt hk)) := by
-  cases' l with hd tl
-  · contradiction
-  · have : k ≤ tl.length := by
-      refine' Nat.le_of_lt_succ _
-      simpa using hk
-    rcases this.eq_or_lt with (rfl | hk')
-    · simp [nthLe_append_right le_rfl, nthLe_cons]
-    · simp [nthLe_append _ hk', length_cons, Nat.mod_eq_of_lt (Nat.succ_lt_succ hk'), nthLe_cons]
-#align list.nth_le_rotate_one List.nthLe_rotate_one
+-- porting note: new lemma
```

Comment:
This isn't a new lemma, it's `#align`ed...

Reply:
Copy+paste error.

Reply:
```suggestion
```